docs/fast_data/monitoring/overview.md

```diff
@@ -6,38 +6,131 @@ sidebar_label: Overview
-
-* [Kafka Exporter](https://github.com/danielqsj/kafka_exporter)
-* [MongoDB Query Exporter](https://github.com/raffis/mongodb-query-exporter)
-
-:::caution
-If you are planning to use [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) as a monitoring system for your project, you can automatically generate and configure the `servicemonitor.yml` files for all the monitoring services using their [Metrics card](/development_suite/api-console/api-design/microservice-monitoring.md).
-:::
-
-## Grafana Dashboards Setup
-
-We assume you already have Grafana up and running, and you are familiar with its basics.
-
-There are three Dashboards ready to use:
-
-1. [Consumer Groups](/fast_data/monitoring/dashboards/consumer_groups.md)
-2. [Projection Changes](/fast_data/monitoring/dashboards/projection_changes.md)
-3. [Single Views](/fast_data/monitoring/dashboards/single_views.md)
-
-More details on what information they provide in their specific pages.
-
-## Putting It All Together
-
-Once your Dashboards are ready, it can be very handy to view them directly inside the console. To do that, you can visit the `Dashboards` section of your project and add the Dashboards you just created to it, as explained [here](/development_suite/monitoring/dashboard.md#add-a-dashboard).
+The Grafana Dashboards make use of [Prometheus](https://prometheus.io/) metrics, which means that you should enable the monitoring of your project (for example, using [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator)) and export the metrics of interest.
+
+If you are planning to use Prometheus Operator as a monitoring system for your project, you can automatically generate and configure the `servicemonitor.yml` files for all the monitoring services using their [Metrics card](/development_suite/api-console/api-design/microservice-monitoring.md).
+
+## Metrics
+
+Metrics are a set of categorized data generally exposed through a `/-/metrics` http GET route which reveals all the Prometheus metrics exposed by each service.
+
+### Fast Data services metrics
+
+Each Fast Data service comes with its set of metrics exposed through an http route. To expose them you only need to enable the `Enable metrics gathering` switch that you can find in each microservice's page in the console.
+
+![Enable metrics toggle in microservice page from the console](../img/monitoring-enable-metrics-toggle.png)
+
+### MongoDB and Apache Kafka metrics
+
+For the dashboards related to MongoDB and Apache Kafka metrics you will need to setup the [Kafka Exporter](https://github.com/danielqsj/kafka_exporter) and [MongoDB Query Exporter](https://github.com/raffis/mongodb-query-exporter) services. Conveniently, you can create them all at once by going to the Applications section of your console and creating a new `Fast Data Monitoring` Application.
```

Comment:
This information is specified in each dashboard details page and since you have go one by one to configure them I don't think the user is going to miss this info. I would not put it here just to avoid having duplicated information lying around that makes things harder to refactor and easier to be outdated